tests/test_warc_to_zim.py

```diff
@@ -21,6 +21,11 @@
 from warc2zim.utils import get_record_url
 
 TEST_DATA_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+# special data dir for WARC files which are not supposed to be ran in the
+# `test_all_warcs_root_dir` test
+TEST_DATA_SPECIAL_DIR = os.path.join(
```

Comment:
???? it's not used

Reply:
and why are we using os.path in this module?

Reply:
The fact it is not used is a mistake. The fact it is using os.path is a legacy. I've fixed both in https://github.com/openzim/warc2zim/pull/347/commits/ddbf74e81b3b16eca8ada90326af187759be2b6c